src/app/common/hooks/use-media-query.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from 'react';
+
+import { BreakpointToken, token } from 'leather-styles/tokens';
+
+function useMediaQuery(query: string) {
+  const [matches, setMatches] = useState(false);
+
+  useEffect(() => {
+    const media = window.matchMedia(query);
+    if (media.matches !== matches) {
+      setMatches(media.matches);
+    }
```

Comment:
maybe just setMatches(media.matches);? do we really need this if check?

Reply:
Let me check if this will work. I just restored this file as I had deleted it in [this PR ](https://github.com/leather-wallet/extension/pull/4655)

Reply:
I'll do this in a follow up.